core/src/main/scala/org/apache/spark/ui/JettyUtils.scala

```diff
@@ -186,6 +188,67 @@ private[spark] object JettyUtils extends Logging {
     contextHandler
   }
 
+  /** Create a handler for proxying request to Workers and Application Drivers */
+  def createProxyHandler(
```

Comment:
I might have overlooked the tests for other handlers, can you point me to them if there is any. I can add test for this handler too then.